packages/flex-plugin-e2e-tests/src/tests/step005.ts

```diff
@@ -12,13 +14,26 @@ const testSuite: TestSuite = async ({ scenario }: TestParams): Promise<void> =>
     from: /This is a dismissible demo component.*/,
     to: scenario.plugin.componentText,
   });
+
+  const envVariableValue = `close-${Date.now()}`;
+
+  writeFileSync(`${scenario.plugin.dir}/.env`, `FLEX_APP_ENVIRONMENT_TEST=${envVariableValue}`);
+
+  await replaceInFile({
+    files: joinPath(scenario.plugin.dir, 'src', 'components', 'CustomTaskList', `CustomTaskList.${ext}`),
+    from: /(close(?!-))|(close-{process.env.FLEX_APP_ENVIRONMENT_TEST})/,
+    to: `close-{process.env.FLEX_APP_ENVIRONMENT_TEST}`,
```

Comment:
Let's change to this to `{process.env.FLEX_APP_ENVIRONMENT_TEST}`, - otherwise the text is going to be become `close-close-Date`